src/cli/backup/backup.go

```diff
@@ -279,6 +279,10 @@ func genericDeleteCommand(
 		return nil
 	}
 
+	if flags.RunModeFV == flags.RunModeFlagTest {
```

Comment:
I'm not sure if this is the best approach. Maybe we should split adding flags and the use of `utils.AddCommand`?

Reply:
It's not the best approach, agreed.  But it's also the pattern we've established in other areas of code.  Let's move this PR forward and we can address the pattern separately.

Reply:
Can we create an issue to track this please? As an alternative, I feel we could add some lightweight integration tests instead, on an empty mailbox/drive/sp/groups.

Reply:
I don't know if we should be adding integrations test for these, as that feels like adding unnecessary complexity just to test availability of flags.

Reply:
Agreed, we should avoid integration tests for this purpose.